charts/spire/charts/spire-server/README.md

```diff
@@ -28,6 +28,61 @@ A Helm chart to install the SPIRE server.
 
 * <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Tornjak
+
+Tornjak is the UI and Control Plane for SPIRE [https://github.com/spiffe/tornjak](https://github.com/spiffe/tornjak) and it is composed of two components:
+
+* Backend (this chart) - Tornjak APIs that extend SPIRE APIs with Control Plane functionality
+* [Frontend](../tornjak-frontend/README.md) - Tornjak UI
+
+When Tornjak is enabled, it can be reached in one of the three connection types:
+
+* HTTP - simple HTTP connection recommended for demonstration purpose only. It should not be use for production.
+* TLS - enables Transport Layer Security (TLS) with HTTPS communication. Tornjak client can validate Tornjak server using TLS Certificate (see instructions below)
+* mTLS - mutual TLS. Both, Tornjak client and the server can validate each other using provided certificates (see instructions below)
```

Comment:
Here a suggestion to update the docs in such way we should also implement this feature to improve the best UX (User Experience) for both the chart installation as well usage of Tornjak itself.

```suggestion
When Tornjak is enabled, it is exposed on both http and https (if TLS server certs are configured). Tornjak handles a permanent redirect from `http` to `https` to ensure users always use the https endpoint.

In addition to that you can configure a `client certificate authority`, this will make Tornjak backend verify Client certificates signed by this authority to enable mTLS authentication.

> **Warning**: For production ensure to configure the certificates including CA, to protect Tornjak from unauthorized access.
```

Reply:
tls is highly recommended. mtls is more of an authentication feature. When oidc is implemented, that may be preferred to be used over mtls in a lot of cases, so we shouldn't necessarily strongly recommend the user ca part.

Reply:
Agree, but for now it is the only option and therefore the current recommendation until we have OIDC available.

Reply:
Yeah. I guess we can amend it once that's implemented to recommend specifying client CA, or OIDC at that point.